integration-libs/opf/base/core/utils/opf-payment-utils.ts

```diff
@@ -19,6 +19,6 @@ export function getBrowserInfo(
     screenWidth: nativeWindow?.screen?.width,
     userAgent: nativeWindow?.navigator?.userAgent,
     originUrl: nativeWindow?.location?.origin,
-    timeZoneOffset: new Date().getTimezoneOffset(),
+    timezoneOffset: new Date().getTimezoneOffset(),
```

Comment:
please ignore, it is to be compatible with test server, will be reverted once Bamboo will fix it.
ref: [CXSPA-4152](https://jira.tools.sap/browse/CXSPA-4152)
OPF Hosted-fields: revert timezoneOffset property typo